generator/simplecpp/README.txt

```diff
@@ -0,0 +1,5 @@
+This code is taken from https://github.com/danmar/simplecpp, version 1.2.0
+
+The code was released under the 0BSD license (see LICENSE file).
+
+The code is lightly patched (see do_not_stop_on_error.patch).
```

Comment:
Forgot - where is `do_not_stop_on_error.patch`?

Reply:
Forgot to add this. Fork didn't show the patch file.